### PR TITLE
Add text for the "rename instances" part

### PR DESCRIPTION
**Pull request by rawfiner**

### CODE REVIEW

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md [hunk 1]
```diff
@@ -823,7 +823,17 @@ TODO
 
 ### Ability to rename module instances
 
-TODO
+It is now possible to set a name to any particular instance.
+This is useful to remember what does each instance, when we have several instances of a module.
+For example, if you use two instances of denoise profile, one for chrominance noise, and one for luminance noise, you may set the name of the first one as "chroma", and the name of the second one as "luma".
+The steps for setting the name of the chroma instance are described bellow:
+* first, click on the "multiple instance action" button
```
Comment (moy) [inline content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md:946]: You need empty lines between items, or they will be considered as italic text (between `*`) on the website.

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md [hunk 2]
```diff
@@ -823,7 +823,17 @@ TODO
 
 ### Ability to rename module instances
 
-TODO
+It is now possible to set a name to any particular instance.
```
Comment (moy) [inline content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md]: Maybe give the motivation first:

When using multiple instances of the same module for different purposes, it is often hard to remember which instance serves which purpose. darktable now allows giving a name to each instance to simplify this.

?

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md (hunk 1 above)
Comment (rawfiner) [reply content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md:946]: Ok, I'll do that

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md (hunk 2 above)
Comment (rawfiner) [reply content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md]: Good idea, it sounds better like this :-)

**Comment by rawfiner** [comment]
I will make these changes tomorrow (probably), I am not on the computer anymore for tonight. Thanks for the review!

**Comment by moy** [comment]
Cc: @edgardoh, the author of the corresponding pull-request. If you have time to proof-read and comment ...

**Comment by rawfiner** [comment]
Should be better now

**Comment by rawfiner** [comment]
(rebased on master)

**Comment by moy** [comment]
I'm merging this, but comments are still welcome e.g. at https://github.com/darktable-org/dtorg/pull/25

Thanks @rawfiner .

**Comment by edgardoh** [comment]
Looks good to me, maybe adding something on how the name interacts with the copy/paste. That's better described in the manual, but the copy/paste will create a new instance or replace a module based on the name, so changing it will affect the result of a past action.

**Comment by moy** [comment]
FYI, I reworked the explanations to have the steps numbered and corresponding to the numbers on the screenshot:

http://matthieu-moy.fr/tmp/2018/wip-dt/2018/12/darktable-26/#rename

(I didn't manage to get the list properly numbered from 2 so I had to edit the PNG to start with step 0 ;-) )

**Comment by moy** [comment]
> maybe adding something on how the name interacts with the copy/paste

Indeed, that was the point of the section right after. The last TODO in the article except for the introduction ;-).

**Pull request merged**